README.md

```diff
@@ -6,3 +6,38 @@
 -   include Profit sharing among marketpalce, theatreOwner and moviemakers in `buyTicket` function
 -   write the fuzzTests
 -   Optional: Integrate with BE and FE
+
+
+
+---
+
+## Inv1nc Ideology
+
+1. Profit Sharing Among Marketplace, Theater Owner, and Movie Makers
+- pros: fair distribution, automation
+- cons: higher gas cost, increased complexity
+
+2. Only Paying the Theater Owner (Assuming Movie Makers Are Already Paid)
+- pros: simplicity, low gas costs
+- cons: assuming movie makers already paid
```

Comment:
It makes more centralized and profit sharing mechanism makes more decentralized and it just require extra two txs.